azure-pipelines.yml

```
@@ -57,9 +79,24 @@ stages:
           ${{ if ne(variables['System.TeamProject'], 'public') }}:
             name: NetCore1ESPool-Internal
             demands: ImageOverride -equals Build.Server.Amd64.VS2017
+        ${{ if eq(variables.runCodeQL3000, 'true') }}:
+          # Component governance and SBOM creation are not needed here. Disable what Arcade would inject.
+          disableComponentGovernance: true
+          enableSbom: false
         variables:
         - name: _HelixBuildConfig
           value: $(_BuildConfig)
+        - ${{ if eq(variables.runCodeQL3000, 'true') }}:
```

Current asplabs-ci builds run CG in public. Will also uselessly auto-inject CG in internal builds, only to be skipped because the Arcade task runs first.
```suggestion
        # Rely on task Arcade injects, not auto-injected build step.
        - skipComponentGovernanceDetection: true
        - ${{ if eq(variables.runCodeQL3000, 'true') }}:
```

---

I'll commit this here since it's nice to have, but I haven't been actively looking for things like this in the other aspnet/* repos since it's unrelated to CodeQL